hub_content/policy_packs/azure/app_service/detect_disabled_http_traffic_for_webapp/policies.tf

```
@@ -0,0 +1,19 @@
+# Azure > AppService > WebApp > Approved
+resource "turbot_policy_setting" "azure_appservice_webapp_approved" {
+  resource = turbot_smart_folder.pack.id
+  type     = "tmod:@turbot/azure-appservice#/policy/types/webAppApproved"
+  value    = "Check: Approved"
+  # value    = "Enforce: Stop unapproved"
+  # value    = "Enforce: Stop unapproved if new"
+  # value    = "Enforce: Delete unapproved if new"
+}
```

```suggestion
  value    = "Check: Enabled"
  # value    = "Enforce: Enabled"
```
We can just keep these 2, check with @SumitPopat